projects/sentinel-trader-bot/index.js

```diff
@@ -0,0 +1,18 @@
+const { PublicKey } = require("@solana/web3.js");
+const { getConnection, decodeAccount, } = require("../helper/solana");
+
+async function tvl() {
+  const connection = getConnection()
+  const mint = await connection.getAccountInfo(new PublicKey('GVeaBeaHZDJHji4UTzaPJgB1PRiVeCV2EaArjYyiwNdT'));
+  const mintInfo = decodeAccount('mint', mint)
+  return {
+    'SENTBOT': mintInfo.supply.toString() / (10 ** mintInfo.decimals)
```

Comment:
This seems like its measuring mcap instead of TVL. Please count the value of tokens locked in the protocol by users